apps/wpcom-block-editor/src/wpcom/features/use-classic-block-guide.js

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/**
+ * External dependencies
+ */
+import domReady from '@wordpress/dom-ready';
+import ReactDOM from 'react-dom';
+import { useState } from '@wordpress/element';
+import { Guide } from '@wordpress/components';
+import { useDispatch } from '@wordpress/data';
+import url from 'url';
+/* eslint-enable import/no-extraneous-dependencies */
+
+const parsedEditorUrl = url.parse( window.location.href, true );
+
+const ClassicGuide = () => {
+	const [ isOpen, setOpen ] = useState( true );
+
+	const closeGuide = () => setOpen( false );
+
```

Comment:
just added to test if we can use useDispatch to toggle the standard welcome guide in this context - seems to work